libs/component-library/src/lib/button/button.tsx

```diff
@@ -52,10 +57,28 @@ const FxButton = ({
   onPressOut,
   variant,
   size,
+  iconLeft,
```

Comment:
Will we have a right icon? Worth adding that in this PR?

Reply:
Sure, I can add that quick

Reply:
done